### DISCUSSION TITLE
Bagaimana cara menggunakan Panel API 'active' => false ?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#219] [Q&A]
![Screenshot 2022-05-25 233330](https://user-images.githubusercontent.com/96925342/170314960-f477ca3b-a93f-42d1-9610-83e9a3d91aa4.jpg)
Kang mau tanya, bagaimana cara menggunakan Panel API `'active' => false`?, saat panel input toggle active, panel lainnya akan ikut active.

`'active' => false` ini saya taruh disetiap panel input lainnya. ketika panel input toggle (sebagai on/off) di ubah ke on/active, panel input berubah jadi `'active' => true`

**Comment by taufik-nurrohman** [comment]
Hebat banget mas sudah bisa menguasai API panel kontrol padahal dokumentasinya belum ada HEHEHEHEHE :+1: :+1: :+1:

Properti `'active' => false` di PHP sebenarnya hanya bertugas untuk menambahkan kelas `is:active` atau `not:active` pada bagian tertentu pada komponen. Selain itu, properti tersebut juga digunakan untuk mengeset atribut `disabled` pada element formulir dalam kondisi tertentu. Kalau mas mau belajar bisa tuh pakai JavaScript untuk melakukan _toggle_ kelas `is:active` dan `not:active` pada elemen target, dan juga _toggle_ atribut `disabled` pada element formulir yang ditargetkan.

**Comment by taufik-nurrohman** [comment]
https://jsfiddle.net/taufik_nurrohman/da824w37/

**Comment by taufik-nurrohman** [comment]
https://github.com/mecha-cms/x.panel/blob/808f04236828c67729f564ac50c495fe6f232e80/engine/f/type.php#L622-L628

https://github.com/mecha-cms/x.panel/blob/808f04236828c67729f564ac50c495fe6f232e80/engine/f/type.php#L644

**Comment by yuliakbard** [comment]
untuk penggunaan ke php langsung gak ada ya kang?

```
        $_['lot']['desk']['lot']['form']['lot'][1]['lot']['tabs']['lot']['jobPosting']['lot']['fields']['lot']['job_posting'] = [
            'title' => 'Job Posting',
            'type' => 'field/toggle',
            'description' => 'Aktifkan Job Posting',
            'name' => 'page[job_posting]',
            'value' => $page['job_posting'],
            'stack' => 10
        ];
        $_['lot']['desk']['lot']['form']['lot'][1]['lot']['tabs']['lot']['jobPosting']['lot']['fields']['lot']['segera'] = [
            //   'active' => $page['job_posting'] == $is_active ? true : false, // kondisional yang mirip/ serupa seperti ini ada gak kang? //
            'title' => 'Segera',
            'type' => 'field/toggle',
            'name' => 'page[segera]',
            'value' => $page['segera'],
            'stack' => 10.2
        ];
```

saya pikir bakal ada kondisional yang seperti ini/ atau serupa. hehehe

**Reply by taufik-nurrohman** [reply]
~~~ php
[
    'active' => !empty($page['job_posting'])
]
~~~
